CHANGELOG.md

```diff
@@ -1,14 +1,44 @@
 # App Center SDK for .NET Change Log
 
-## Version 1.12.0 (Under active development)
+## Version 1.12.0 
+
+### AppCenter
+
+#### Android
+* **[Fix]** Do not delete old logs when trying to add a log larger than the maximum storage capacity.
+* **[Fix]** Fix minimum storage size verification to match minimum possible value.
+* **[Fix]** Fix disabling logging of network state changes according to AppCenter.getLogLevel.
+* **[Fix]** Fix logs duplication on unstable network.
+
+#### iOS
+* **[Fix]** Do not delete old logs when trying to add a log larger than the maximum storage capacity.
+* **[Fix]** Fix minimum storage size verification to match minimum possible value.
+* **[Fix]** Fix reporting carrier information using new iOS 12 APIs when running on iOS 12+.
+* **[Fix]** Fix a memory leak issue during executing SQL queries.
+* **[Fix]** Fix a keychain permission issue on macOS applications.
```

Comment:
Remove this line, we don't support MacOS on Xamarin

Reply:
this was not updated, this line must be removed . i didnt get this . here what exactly need to changed